clients/cpp-console/src/common/mainAudio.cpp

```diff
@@ -268,42 +287,64 @@ int main(int argc, char** argv)
 
         auto continue_multiturn = activity.value<string>("inputHint", "") == "expectingInput";
 
-        uint32_t total_bytes_read = 0;
         if (event.HasAudio())
         {
-            log_t("Activity has audio, playing synchronously.");
+            log_t("Activity has audio, playing asynchronously.");
 
-            // TODO: AEC + Barge-in
-            //log_t("Pausing KWS during TTS playback");
-            //PauseKws();
+            if(!bargeInSupported)
+            {
+                log_t("Pausing KWS during TTS playback");
+                PauseKws();
+            }
 
             auto audio = event.GetAudio();
             int play_result = 0;
-
+    
+            uint32_t total_bytes_read = 0;
             if(volumeOn && player != nullptr){
-                play_result = player->Play(audio);
+                
+                //if we are expecting more input and have audio to play, we will want to wait till all audio is done playing before
```

Comment:
Commands start with capital letter.
Space between // and the comment

Reply:
and elsewhere

Reply:
fixed